api/src/controllers/changes.js

```diff
@@ -192,37 +169,34 @@ const restartNormalFeed = feed => {
 };
 
 const getChanges = feed => {
-  const options = {
-    since: feed.req.query && feed.req.query.since || 0,
-    // PouchDB batches requests using this value as a limit and keeps issuing requests until the result
-    // set length is lower than the limit.
-    batch_size: MAX_DOC_IDS + 1
```

Comment:
Removing this parameter causes PouchDB to batch the requests with a limit of 25 (with actual `limit` param, not by batching actual docIDs) which yields incomplete results (as discussed in Slack, due to CouchDB returning an incorrect `last_seq`).

Reply:
Fixed.